common/messages.go

```diff
@@ -40,6 +40,9 @@ type View struct {
 	Name            string   `json:"name" msgpack:"name"`
 	LayoutType      string   `json:"layout_type" msgpack:"layout_type"`
 	DefaultRequests []string `json:"default_requests" msgpack:"default_requests"`
+
+	// optional UI view settins: more exist on FE "LayoutOptions" but aren't all exposed via schema
+	FieldColumns int `json:"field_columns" msgpack:"field_columns"`
```

Comment:
@oklex looks good, but if you want to omit the return values if there is no value lets do this and that should ensure we don't return a empty value to frontend 

FieldColumns int `json:"field_columns,omitempty" msgpack:"field_columns,omitempty"`